src/expression/visibility.ts

```diff
@@ -0,0 +1,43 @@
+import {createExpression, findGlobalStateRefs} from '.';
```

Comment:
Should this be imported from a more specific location? It's weird to see import from "."...